src/aks-preview/HISTORY.md

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+0.1.1
++++++
+* Change default agentType to VMAS
```

Comment:
This is a breaking change. Please indicate so in the history with `BREAKING CHANGE:` and bump the minor version.